### ISSUE TITLE
LoadImageD unwanted fip in PNG files

### ISSUE THREAD

**Issue by alessandrocasella** [Project-MONAI/MONAI#3976]
**Describe the bug**
When using LoadImageD on dataset with image,label pair both are loaded and flipped on both axis

**To Reproduce**
Normal training or inference pipeline 
val_imtrans = Compose([
        LoadImaged(["image", "label"], reader='pilreader', ensure_channel_first=True, image_only=True),
        ScaleIntensityRanged("image",0, 255, 0, 1),
        LabelFilterd("label", 1),
        EnsureTyped(["image", "label"],),
        ToDeviced(["image", "label"],device=device)
    ])
val_ds = CacheDataset(val_dict, val_imtrans, cache_rate=1.0, copy_cache=True, num_workers=4)
val_loader = ThreadDataLoader(val_ds, batch_size=1, num_workers=0)
roi_size = (224, 224)
sw_batch_size = 16
val_pred = sliding_window_inference(val_images, roi_size, sw_batch_size, model,overlap=0.75, sw_device=device, device=device)
            
**Expected behavior**
No rotation applied if not requested

**Screenshots**
N/A

**Environment**

Python 3.7
Pytorch 1.10
Cuda 11.6
Monai 0.81 from github (updated on march, 1st)

**Comment by wyli** [comment]
that's done so that the image reader backends follow the same convention... https://github.com/Project-MONAI/MONAI/blob/459b1885de2c3415bc784fbb0382a4165aad63b4/monai/data/image_reader.py#L681-L682
